CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Maintainers
+This repo is currently maintained by Ken Liao (@yknl). 
+
+# Pull Request Reviewers
+Hank Stoever (@hstove)
+Aaron Blankstein (@kantai)
+Matthew Little (@zone117x)
+
+# Submitting Pull Requests
+This repository uses the [git flow branching mode](http://nvie.com/posts/a-successful-git-branching-model/).
+
+The latest released code as deployed to npm is in `master` and the latest delivered development
+changes for the next release are in `develop`.
+
+We use the [git-flow-avh](https://github.com/petervanderdoes/gitflow-avh) plugin.
+
+Pull request requirements:
+
+1. Describe exactly what the goal of the PR is (and link to any relevant issues)
+2. Describe how that goal was achieved through the submitted implementation.
+3. The code must be lint-free and pass the unit tests by running `$ npm run test`
+4. The code should contain [flow static type annotations](https://flow.org)
+5. Contain tests that cover any and all new functionality or code changes.
+6. Describe how the new functionality can be tested manually.
+7. Document any new features or endpoints, and describe how developers would be expected to interact with them.
```

Comment:
We should also require that the build the documentation and check it. Or alternatively, build the docs as part of the automated github checks. Then, require PR reviewers to look at this check.